README.md

```diff
@@ -46,6 +46,9 @@ export GITHUB_STATUS_CONTEXT="codebuild/pr"  # Tag associated with the status ch
 export GITHUB_BUILD_EVENTS="pr_state"  # Comma-separated string of build events, supports "pr_state" and "pr_comment"
 export GITHUB_BUILD_USERS=""  # Comma-separated string of GitHub users authorized to build from PR comments
 export GITHUB_BUILD_COMMENT="go codebuild go"  # Case-insensitive string that will start a build from a PR comment
+export CB_EXTERNAL_BUILDSPEC=false # If "false", webhook will not have CodeBuild submit source version in buildspec.yml request (useful if in 2 different repos or you want to use the most recent commit)
```

Comment:
The default should be "true" to retain prior behavior.

Also, I think the description here is too much detail. Should be sufficient to say something like "Set to false if the repo for the CodeBuild project itself is different from the repo where the webhook will be created"

Reply:
Oops, we flipped the logic, "false" is the correct default. "Set to _true` if ..."